includes/classes/admin/class-admin.php

```diff
@@ -441,6 +451,9 @@ public function pilopress_dashboard() {
          */
         public function display_pip_navbar() {
 
+            // Hide ACF top navbar
+            add_filter( 'acf/admin/toolbar', '__return_false' );
```

Comment:
We will need to remove this line (probably some out of sync git things but : https://github.com/Pilot-in/PiloPress/commit/ff1e79e67cf9abd6ce5e1fce4b087d8a094f44da#diff-480e4b52bf9e25a6024e4d81e3ca1ef5ab570dfca5f5255c734872914acfec2c)